tests/e2e/Services/Functions/FunctionsCustomServerTest.php

```diff
@@ -303,6 +299,41 @@ public function testCreateDeployment($data):array
 
         // Wait for deployment to build.
         sleep(15);
+
+        /**
+         * Test for Large Code File SUCCESS
+         */
+        $source = realpath(__DIR__ . '/../../../resources/functions/php-large.tar.gz');
```

Comment:
We have a easier way to do this
https://github.com/appwrite/appwrite/blob/feat-functions-refactor/tests/e2e/Services/Functions/FunctionsCustomServerTest.php#L285-L295

We don't keep tar files in the repo anymore. The test suite creates the tar files from the source code during runtime. 

So you can remove this file as well
tests/resources/functions/php-large.tar.gz